noxfile.py

```diff
@@ -10,7 +10,8 @@
 def tests_impl(
     session: nox.Session,
     extras: str = "socks,brotli,zstd",
-    byte_string_comparisons: bool = True,
+    # hypercorn dependency h2 compares bytes and strings
```

Comment:
Can we create an issue on h2 for this? Would be good to keep this one enabled.

Reply:
Thanks, I added a link to https://github.com/python-hyper/h2/issues/1236 in https://github.com/urllib3/urllib3/pull/3190/commits/be8f4e786ef67ac3c77e122dd9479da947901b79. Seems difficult to fix at first glance.